ocs_ci/framework/testbed.py

```diff
@@ -0,0 +1,176 @@
+import logging
```

Comment:
Should we put the file under ocs_ci/framework/? the content of this file doesn't match with the other files under this dir